llm/dbrx/dbrx.yaml

```diff
@@ -0,0 +1,84 @@
+# Serving Databricks DBRX on your own infra.
+#
+# Usage:
+#
+#  HF_TOKEN=xxx sky launch vllm.yaml -c dbrx --env HF_TOKEN
+#
+# curl /v1/chat/completions:
+#
+#   IP=$(sky status --ip dbrx)
+#   curl $IP:8081/v1/models
+#   curl http://$IP:8081/v1/chat/completions \
+#     -H "Content-Type: application/json" \
+#     -d '{
+#       "model": "databricks/dbrx-instruct",
+#       "messages": [
+#         {
+#           "role": "system",
+#           "content": "You are a helpful assistant."
+#         },
+#         {
+#           "role": "user",
+#           "content": "Who are you?"
+#         }
+#       ]
+#     }'
+#
+# Chat with model with Gradio UI:
+#
+#   Running on local URL:  http://127.0.0.1:8811
+#   Running on public URL: https://<hash>.gradio.live
+
+envs:
+  MODEL_NAME: databricks/dbrx-instruct
+  HF_TOKEN: <your-huggingface-token>  # Change to your own huggingface token, or use --env to pass.
+
+service:
+  readiness_probe: /v1/models
```

Comment:
Would be better to use actual request as readiness probe as `/v1/models` not necessarily show the healthiness.

https://github.com/skypilot-org/skypilot/blob/49fd20880ec6ebfa8fc6e4c9ed9ff9f70384a215/llm/mixtral/serve.yaml#L8-L14